Vagrantfile

```diff
@@ -0,0 +1,71 @@
+# -*- mode: ruby -*-
```

Comment:
Correct. Be sure that it is excised from the commit history instead of just deleting the file.

https://help.github.com/articles/removing-sensitive-data-from-a-repository/